shiksha-migration/update-intersted-to-join.js

```
@@ -0,0 +1,102 @@
+                console.log(`[UPDATE] ⚠️  Skipped UserID=${userId} — not found in destination DB`);
+                skipped++;
+            }
+        }
+
+        console.log('\n=== SUMMARY ===');
+        console.log(`  Total source records : ${sourceResult.rows.length}`);
+        console.log(`  Updated              : ${updated}`);
+        console.log(`  Skipped (not found)  : ${skipped}`);
+
+    } catch (error) {
+        console.error('[UPDATE] ❌ Error:', error.message);
```

ESLint

[error] Delete `····`

(prettier/prettier)